xml/bp_troubleshooting_cephfs.xml

```diff
@@ -271,4 +271,382 @@ sudo zypper in ceph-common
+           <entry>0x20000000000000</entry>
+         </row>
+         <row>
+           <entry>OSD_HITSET_GMT</entry>
+           <entry>54</entry>
+           <entry>0x40000000000000</entry>
+         </row>
+         <row>
+           <entry>HAMMER_0_94_4</entry>
+           <entry>55</entry>
+           <entry>0x80000000000000</entry>
+         </row>
+         <row>
+           <entry>NEW_OSDOP_ENCODING</entry>
+           <entry>56</entry>
+           <entry>0x100000000000000</entry>
+         </row>
+         <row>
+           <entry>MON_STATEFUL_SUB</entry>
+           <entry>57</entry>
+           <entry>0x200000000000000</entry>
+         </row>
+         <row>
+           <entry>MON_ROUTE_OSDMAP</entry>
+           <entry>57</entry>
+           <entry>0x200000000000000</entry>
+         </row>
+         <row>
+           <entry>OSDSUBOP_NO_SNAPCONTEXT</entry>
+           <entry>57</entry>
+           <entry>0x200000000000000</entry>
+         </row>
+         <row>
+           <entry>SERVER_JEWEL</entry>
+           <entry>57</entry>
+           <entry>0x200000000000000</entry>
+         </row>
+         <row>
+           <entry>CRUSH_TUNABLES5</entry>
+           <entry>58</entry>
+           <entry>0x400000000000000</entry>
+         </row>
+         <row>
+           <entry>NEW_OSDOPREPLY_ENCODING</entry>
+           <entry>58</entry>
+           <entry>0x400000000000000</entry>
+         </row>
+         <row>
+           <entry>FS_FILE_LAYOUT_V2</entry>
+           <entry>58</entry>
+           <entry>0x400000000000000</entry>
+         </row>
+         <row>
+           <entry>FS_BTIME</entry>
+           <entry>59</entry>
+           <entry>0x800000000000000</entry>
+         </row>
+         <row>
+           <entry>FS_CHANGE_ATTR</entry>
+           <entry>59</entry>
+           <entry>0x800000000000000</entry>
+         </row>
+         <row>
+           <entry>MSG_ADDR2</entry>
+           <entry>59</entry>
+           <entry>0x800000000000000</entry>
+         </row>
+         <row>
+           <entry>OSD_RECOVERY_DELETES</entry>
+           <entry>60</entry>
+           <entry>0x1000000000000000</entry>
+         </row>
+         <row>
+           <entry>CEPHX_V2</entry>
+           <entry>61</entry>
+           <entry>0x2000000000000000</entry>
+         </row>
+         <row>
+           <entry>RESERVED</entry>
+           <entry>62</entry>
+           <entry>0x4000000000000000</entry>
+         </row>
+       </tbody>
+     </tgroup>
+   </table>
+   <para>
+     A possible solution to allow old kernel clients to mount a recent &cephfs;
+     is to modify the CRUSH profile.  For example, to fix the problem shown
+     above, the following command can be used:
+   </para>
+<screen>
+&prompt.cephuser;ceph osd crush tunables hammer
+</screen>
+    <para>
+      This will adjust the CRUSH profile to the behaviour it had for the Hammer
+      (0.94) release.  Note however that this is not the optimal behaviour for
```

Comment:
If this value it a text string, then without considerable prior knowledge, it is not possible to work out what values can be used. 

Therefore, I think we should specify a list of permissible values and what they mean. I think that probably means another table, I am afraid.

I would suggest at a minimum, the following values:
col1: name of release (e.g. "hammer")
col2: version number or date of release (or some other value allowing them to be ordered)
col3: SES version that equates to that release

The columns need not be in that order, but I don't think we can just leave people to try to Google what word they need to insert here to turn certain features on or off.

Reply:
Ok, makes sense.  I'll add a table or a list for this.  However, I don't think the corresponding SES version  is required because AFAIK the first release we support CephFS (SES5) is more recent than any value accepted for the profiles.  Let's see what I can come up with.

Reply:
OK, fair, but it might be useful reference for customers who have not memorised a list of Ceph releases. Many people are too busy for things like this.

In case it is helpful:
https://docs.ceph.com/en/latest/releases/
https://docs.ceph.com/en/latest/releases/archived-index/
I think we don't need all the point-releases -- just those where the name changes, at the most.

Reply:
In fairness, we _assume_ the customer is familiar with Ceph and their release names. I don't see this as an issue.